README.md

```diff
@@ -125,7 +126,7 @@ jobs:
             "results/"
       - name: Upload SARIF file
         uses: github/codeql-action/upload-sarif@v2
-        if: always()
+        if: ${{ steps.orcasecurity.outputs.exitcode != 1 }}
```

Comment:
What if we will add more exit codes to indicate unexpected errors?
I'm not sure we want the customer to use the exact exitcode here. Maybe it's better to change the output to a boolean value -  maybe `completed` or `error` and to calculate the value in the entrypoint